project_event/models/project_project.py

```diff
@@ -123,8 +123,18 @@ def get_message_body(self, action):
         return switcher.get(action)
 
     def get_message(self, action):
+        message = '<br>'
+        message += _('Event: <br>') + self.name + '<br>'
+        for activity in self.task_ids:
+            message += _('Activity: ') + activity.name + '<br> Tasks:'
```

Comment:
@eilst :  message += _('Activity: ') + activity.name + '<br> Tasks:' is the "Tasks" adding to the message . no standardazied format